supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/models/GcpBatchRuntimeAttributes.scala

```diff
@@ -51,7 +51,6 @@ final case class GcpBatchRuntimeAttributes(cpu: Int Refined Positive,
 object GcpBatchRuntimeAttributes {
 
   val ZonesKey = "zones"
-
   private val ZonesDefaultValue = WomString("us-central1-b")
```

Comment:
For a different PR, but all zones in the region is a better default

Reply:
Yeah, that seems like a weird default